### ISSUE TITLE
Best way to annotate/typedef a callback?

### ISSUE THREAD

**Issue by mmoss** [esdoc/esdoc#53]
Just wanted to say THANKYOU this project is great!

Quick question. What is the best way to write a typedef, or document a callback whose signature is used in several places?

**Comment by h13i32maru** [comment]
@mmoss Hi! Thanks this question!
I suggest the following code.

```javascript
/**
 * @param {myCallback} callback - this is callback.
 */
export function myFunction(callback){}

/**
 * @typedef {function} myCallback
 * @param {number} foo - this is foo.
 * @param {string} bar - this is bar.
 */
```

However, unfortunately ESDoc can not correctly parse this code :sweat: now.
I try to fix this problem. Please wait a little.

**Comment by mmoss** [comment]
No problem! I like the idea, let me know if I can help! :grin:

**Comment by h13i32maru** [comment]
@mmoss I released ESDoc v0.2.3. This ESDoc fixed the problem.
Please try to it :smile:

**Comment by mmoss** [comment]
Awesome! I'll try it out today :v:
